packages/swapper/src/swappers/ChainflipSwapper/endpoints.ts

```diff
@@ -188,23 +189,14 @@ export const chainflipApi: SwapperApi = {
 
     const adapter = assertGetUtxoChainAdapter(step.sellAsset.chainId)
 
-    const sendAddress = await adapter.getAddress({
-      accountNumber: step.accountNumber,
-      // @ts-ignore this is a rare occurence of wallet not being passed but this being fine as we pass a pubKey instead
-      // types are stricter than they should for the sake of paranoia
```

Comment:
Turns out, types were not stricter than they should.

![image](https://github.com/user-attachments/assets/931a6ab6-fb2d-4568-9555-85837523f7da)